tests/e2e/flowtest/test_scc.py

```diff
@@ -0,0 +1,55 @@
+import pytest
+import logging
+
+from ocs_ci.ocs import constants
+from ocs_ci.utility import templating
+from ocs_ci.helpers.helpers import create_resource, wait_for_resource_state
+from ocs_ci.ocs.ocp import OCP
+from ocs_ci.ocs.resources.pod import Pod, get_pods_having_label
+from ocs_ci.ocs.exceptions import ResourceWrongStatusException
+from ocs_ci.framework.pytest_customization.marks import bugzilla, polarion_id
+
+logger = logging.getLogger(__name__)
+
+
+class TestSCC:
+    @pytest.fixture(scope="function")
+    def create_scc(self, request):
+        scc_data = templating.load_yaml(constants.SCC_YAML)
```

Comment:
are we able to reproduce this issue with the same yaml in the build where the issue is found?

Reply:
As of now, I haven't tried to reproduce this issue in the 4.6. But as per https://bugzilla.redhat.com/show_bug.cgi?id=1938647#c14 this supposed to be only verified through regression but to be on safer side.. Ashish had suggested me the steps I'm covering in this automation. And more importantly.. this yaml has the exact specifications of nooba-endpoint scc which is used by noobaa db pods and added to group system:authenticated. This is to make sure while re-spinning nooba db pods, custom scc wont impact the reconcile